makelib/helm.mk

```diff
@@ -113,6 +118,16 @@ helm.promote: $(HELM_HOME)
 	@rm -fr $(HELM_TEMP)
 	@$(OK) promoting helm charts
 
+helm.museum: $(HELM_OUTPUT_DIR)/$(1)-$(HELM_CHART_VERSION).tgz
```

Comment:
what exactly is the `$(1)` in this dependency? this one is a bit unintuitive

Reply:
I think I need to redesign the entire thing. it's not that straightforward that I thought

Reply:
can you expand on that thought @arturrez? I'm not sure I have the full context to understand where you think it needs to go.